src/utils/utils.py

```diff
@@ -0,0 +1,81 @@
+import csv
+import json
+
+
+def jsonify_data(records: list, labelled=False):
+    """
+    Create json string from feedback
+    :return: json string of feedback records
+    """
+    subs = []
+    for i, item in enumerate(records):
```

Comment:
Out of interest, why enumerate here given i isn't used anywhere?

Reply:
removed :)